intel_models/text-detection-0001/description/text-detection-0001.md

```diff
@@ -0,0 +1,42 @@
+# text-detection-0001
+
+## Use Case and High-Level Description
+
+Text detector based on [PixelNet](https://arxiv.org/pdf/1801.01315.pdf) architecture with [MobileNetV2](https://arxiv.org/pdf/1801.04381.pdf) as a backbone for indoor/outdoor scenes.
+
+## Example
+
+![](./text-detection-0001.png)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| F-measure (Harmonic mean of precision and recall on ICDAR2015)| 80.13%                  |
+| GFlops                                                        | 51.256                  |
+| MParams                                                       | 6.747                   |
+| Source framework                                              | TensorFlow              |
+
+## Performance
+Link to [performance table](https://software.intel.com/en-us/openvino-toolkit/benchmarks)
+
+## Inputs
+
+1. name: "input" , shape: [1x3x768x1280] - An input image in the format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - RGB.
```

Comment:
Please check the color order for models.
Also here: https://github.com/opencv/open_model_zoo/blob/1b1581cd4c6bc1435da1aaa23bf693333d055769/intel_models/landmarks-regression-retail-0009/description/landmarks-regression-retail-0009.md#L39

Reply:
cc @snosov1 
@zhiltsov-max when the model is converted to IR, there is a specific flag <reverse_input_channels value="True"/> in the IR's meta-information so the model should be ready to accept input as BGR instead of RGB. This allows to avoid unnecessary color conversions. 
I see this flag for both the models which make me thinking it's actually accepting BGR so the documentation needs to be changed.